content/source/docs/enterprise/before-installing/index.html.md

```diff
@@ -141,6 +141,47 @@ Terraform Enterprise's instance profile serves as default credentials for Terraf
 
 The instance profile of Terraform Enterprise's instance is the operator's responsibility. If you plan to specify any non-default permissions for Terraform Enterprise's instance profile, be aware that Terraform runs might use those permissions and plan accordingly.
 
+#### S3 IAM Policy
```

Comment:
Nitpick. Since we now have two sections at this level it is worth keeping them uniform.

```suggestion
#### S3 Policy
```